packages/backend/src/controllers/orders.controller.ts

```
@@ -1,100 +1,99 @@
-// Args: req.body: { userId: string, offerId: string, totalAmount: number }
-// Returns: 201 + commande créée
-export const createOrder = async (req: Request, res: Response): Promise<void> => {
+/**
+ * Create a new order
+ * @route POST /api/orders
+ * @access Admin only
+ * @param req Express Request
+ * @param res Express Response
+ * @returns 201 + Order | 400 | 500
+ */
+export const createOrder = async (req: Request, res: Response): Promise<Response | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)